From 5218bec0716d132078a416acb15612fc70e55b80 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Tue, 23 Oct 2018 20:14:46 +0300
Subject: [PATCH] ITS [GAUDI PLDM]: fix kernel data name

By mistake image is uzed is commpressed (GZIP) but commpression left as
none
---

diff --git a/board/habanalabs/gaudi/gaudi_pldm.its b/board/habanalabs/gaudi/gaudi_pldm.its
index c4da25a..ab201aa 100644
--- a/board/habanalabs/gaudi/gaudi_pldm.its
+++ b/board/habanalabs/gaudi/gaudi_pldm.its
@@ -12,7 +12,7 @@
 	images {
 		kernel@1 {
 			description = "Gaudi Linux kernel";
-			data = /incbin/("Image.gz");
+			data = /incbin/("Image");
 			type = "kernel";
 			arch = "arm64";
 			os = "linux";
